draft-ietf-quic-recovery.md

```diff
@@ -518,14 +518,20 @@ or when Initial or Handshake keys are discarded. This ensures the PTO is always
 set based on the latest RTT information and for the last sent packet in the
 correct packet number space.
 
+When setting the PTO timer, the ApplicationData packet number space (Section
```

Comment:
Can you clarify what the goal of changing this section is?  I feel it was clearer before, but I suspect there's something you're trying to improve.

Reply:
As pointed out in https://github.com/quicwg/base-drafts/pull/3874#discussion_r452560791, the existing text can be read in two ways, depending on how you interpret "with one exception." This PR rearranges two requirements to different paragraphs to be clear that there is no dependency between the two requirements.